src/ExportPatchesCommand.php

```diff
@@ -26,11 +26,15 @@ public function __invoke(): void {
     if (!$composer_json) {
       return;
     }
-    $dependencies = array_map(
-      fn($dependency) => basename($dependency),
-      array_keys($composer_json['require'])
-    );
-    $dependencies = array_combine($dependencies, $dependencies);
+
+    $dependencies = [];
+    foreach(array_keys($composer_json['require']) as $dependecy) {
+      $dependencies = $dependencies + [basename($dependecy) => $dependecy];
```

Comment:
The result of this complex operation is a regular array set
```suggestion
      $dependencies[basename($dependency)] = $dependency;
```